release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1106,10 +1108,20 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 [id="ocp-4-17-node-bug-fixes_{context}"]
 ==== Node
 
+* Previously, the Container Runtime Config controller did not consider whether a mirror configuration was in use when adding the scope from a `ClusterImagePolicy` CR to /etc/containers/registries.d/sigstore-registries.yaml. As a consequence, image verification failed with a `Not looking for sigstore attachments` message. With this fix, images are pulled from the mirror registry as expected. (link:https://issues.redhat.com/browse/OCPBUGS-36344[*OCPBUGS-36344*])
+
+* Previously, a group ID was not added to the /etc/group within a container when the `spec.securityContext.runAsGroup` attribute was set in the pod spec. With this release, this issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-39478[*OCPBUGS-39478*])
+
+* Previously, because of a critical regression on RHEL 9.4 kernels earlier than `5.14.0-427.26.1.el9_4`, the `mglru` feature had memory management disabled. In this release, the regression issue is fixed so that the `mglru` feature is now enabled in {product-title} {product-version}. (link:https://issues.redhat.com/browse/OCPBUGS-35436[*OCPBUGS-35436*])
+
 [discrete]
 [id="ocp-4-17-node-tuning-operator-bug-fixes_{context}"]
 ==== Node Tuning Operator (NTO)
 
+* Previously, due to an internal bug, the Node Tuning Operator incorrectly computed CPU masks for interrupt and network handling CPU affinity if a machine had more than 256 CPUs. This prevented proper CPU isolation on those machines and resulted in `systemd` unit failures. With this release, the Node Tuning Operator computes the masks correctly. (link:https://issues.redhat.com/browse/OCPBUGS-39164[*OCPBUGS-39164*]) 
```

Comment:
```suggestion
* Previously, due to an internal bug, the Node Tuning Operator incorrectly computed CPU masks for interrupt and network-handling CPU affinity if a machine had more than 256 CPUs. This prevented proper CPU isolation on those machines and resulted in systemd unit failures. With this release, the Node Tuning Operator computes the masks correctly. (link:https://issues.redhat.com/browse/OCPBUGS-39164[*OCPBUGS-39164*]) 
```
We don't backtick systemd in MicroShift because RHEL doesn't, see https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/using_systemd_unit_files_to_customize_and_optimize_your_system/index. Doesn't look like OCP does, either, see https://docs.openshift.com/container-platform/4.16/observability/logging/config/cluster-logging-systemd.html

Reply:
:checklist